cloud-sql/mysql/servlet/src/main/java/com/example/cloudsql/ConnectionPoolContextListener.java

```diff
@@ -47,6 +47,10 @@
       justification = "Necessary for sample region tag.")
   private DataSource createConnectionPool() {
     // [START cloud_sql_mysql_servlet_create]
+    // Note: Connecting to the Cloud SQL Proxy using Unix sockets is not supported in Java.
+    // For Java users, the Cloud SQL JDBC Socket Factory can provide authenticated connections.
+    // See https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory for details.
```

Comment:
The link you provided says it still works.
https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory#connection-via-unix-sockets